llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -1090,6 +1091,36 @@ RISCVTargetLowering::RISCVTargetLowering(const TargetMachine &TM,
         ISD::VECREDUCE_FMINIMUM,
         ISD::VECREDUCE_FMAXIMUM};
 
+    // TODO: support more ops.
```

Comment:
Just to check, this list should be come smaller over time right? Should the comment say something like

```suggestion
    // TODO: Make more of these ops legal.
```